Integrations/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.py

```diff
@@ -0,0 +1,316 @@
+    }
+
+    if access_type:
+        data['AccessType'] = access_type
+
+    if reason:
+        data['Reason'] = reason
+
+    if access_policy_schedule_id:
+        data['AccessPolicyScheduleID'] = access_policy_schedule_id
+
+    if conflict_option:
+        data['ConflictOption'] = conflict_option
+
+    request = create_release_request(str(data))
+    request_id = str(request)
+
+    credentials = get_credentials_request(request_id)
+
+    response = {
+        'RequestID': request_id,
+        'Credentials': credentials
+    }
+
+    ec = {'BeyondTrust.Request(val.AccountId === obj.AccountId)':createContext(response)}
+    return_outputs(tableToMarkdown('The request has been Successful', response), ec, response)
+
+
+def get_credentials_request(request_id):
+
+    suffix_url = '/credentials/' + request_id
+    response = http_request('GET', suffix_url)
+
+    return response
+
+
+# def get_credentials():
+#     """
+#     Retrieves the credentials for an approved and active (not expired) credentials release request.
+#     """
+
+#     request_id = demisto.args().get('request_id')
+#     response = get_credentials_request(request_id)
+
+#     ec = {'BeyondTrust.Request(val.request_id === obj.request_id)': response}
+
+#     return_outputs(tableToMarkdown('BeyondTrust Release Request', response, removeNull=True), ec, response)
+
+
+def check_in_credentials_request(request_id, data):
+
+    suffix_url = '/Requests/' + request_id + '/Checkin'
+    response = http_request('PUT', suffix_url, data=json.dumps(data))
+
+    return response
+
+
+def check_in_credentials():
+    """
+    Checks-in/releases a request before it has expired.
+    """
+    request_id = demisto.args().get('request_id')
+    reason = demisto.args().get('reason')
+
+    data = {'Reason': reason if reason else ''}
+
+    check_in = check_in_credentials_request(request_id, data)
+
+    demisto.results('The release was successfully checked-in/released')
+
+
+def change_credentials_request(account_id, data):
+
+    suffix_url = '/ManagedAccounts/' + account_id + '/Credentials'
+    response = http_request('PUT', suffix_url, data=json.dumps(data))
+
+    return response
+
+
+def change_credentials():
+    """
+    Updates the credentials for a Managed Account, optionally applying the change to the Managed System.
+    """
+    account_id = demisto.args().get('account_id')
+    password = demisto.args().get('password')
+    public_key = demisto.args().get('public_key')
+    private_key = demisto.args().get('private_key')
+    pass_phrase = demisto.args().get('pass_phrase')
+    update_system = demisto.args().get('update_system')
+
+    data = {
+        'AccountId': account_id
+    }
+
+    if password:
+        data['Password'] = password
+
+    if private_key:
+        if public_key and update_system == True:
+            data['PrivateKey'] = private_key
+            data['PublicKey'] = public_key
+        else:
+            demisto.results('Please provide public key')
```

Comment:
dont we want to exit the function after that?
also i think we should return an error here
dont use `Please` please :)